Contentstack.Core/ContentstackClient.cs

```diff
@@ -34,6 +41,8 @@ internal string StackApiKey
             set;
         }
         private ContentstackOptions _options;
+        private HttpClient _httpClient;
+        private string xUserAgent => $"contentstack-delivery-dotnet/{Version}";
```

Comment:
looks like you have copied code from management sdk. Please write your own code

remove delivery from **contentstack-delivery-dotnet**